### PR TITLE
Us 936 allow to user export parameters from front

### PR DESCRIPTION
**Pull request by laurgarn**

### CODE REVIEW

**Review by JacquesBorsenberger — APPROVED** [review]

File: src/main/java/org/gridsuite/study/server/StudyService.java [hunk 1]
```diff
@@ -944,7 +944,7 @@ private Mono<Void> setLoadFlowRunning(UUID studyUuid, UUID nodeUuid) {
             );
     }
 
-    public Mono<Collection<String>> getExportFormats() {
+    public Mono<String> getExportFormats() {
         String path = UriComponentsBuilder.fromPath(DELIMITER + NETWORK_CONVERSION_API_VERSION + "/export/formats")
             .toUriString();
```
Comment (JacquesBorsenberger) [inline src/main/java/org/gridsuite/study/server/StudyService.java:950]: is this variable still used ?

File: src/main/java/org/gridsuite/study/server/StudyService.java [hunk 2]
```diff
@@ -954,10 +954,10 @@ public Mono<Collection<String>> getExportFormats() {
         return webClient.get()
             .uri(networkConversionServerBaseUri + path)
             .retrieve()
-            .bodyToMono(typeRef);
+            .bodyToMono(String.class);
```
Comment (JacquesBorsenberger) [inline src/main/java/org/gridsuite/study/server/StudyService.java:957]: is this variable still used ? (typeRef)

Comment (laurgarn) [reply src/main/java/org/gridsuite/study/server/StudyService.java:954]: Corrected :-)

File: src/test/java/org/gridsuite/study/server/StudyTest.java [hunk 3]
```diff
@@ -1032,6 +1032,10 @@ public void test() throws Exception {
 
         assertTrue(getRequestsDone(1).contains(String.format("/v1/networks/%s/export/XIIDM", NETWORK_UUID_STRING)));
 
+        mockMvc.perform(get("/v1/studies/{studyUuid}/nodes/{nodeUuid}/export-network/{format}?formatParameters=%7B%22iidm.export.xml.indent%22%3Afalse%7D", studyNameUserIdUuid, rootNodeUuid, "XIIDM"))
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/study/server/StudyTest.java:1035]: Maybe use URLEncoder.encode to pass the formatParameters json value ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**